doc.json

```diff
@@ -442,7 +442,99 @@
           "description": "Permission denied, user cannot rate their own article"
           }
         }
+      }
+    },
+      "/likeArticle/:slug": {
```

Comment:
@Dubby20 it's an anti-pattern to use camel case routes. It's proper to use "_" to separate words in routes. If there are other places in the app where this exists, please let me know, so we can change it.

Reply:
I've changed the routes url to use snake_case